cgcnn2/util.py

```
@@ -127,8 +126,14 @@ def cgcnn_test(
             outputs_list.extend(output.cpu().numpy().ravel().tolist())
             cif_ids.extend(cif_id)
 
-    mse = mean_squared_error(targets_list, outputs_list)
-    r2 = r2_score(targets_list, outputs_list)
+    targets_array = np.array(targets_list)
```

Ruff (0.11.9)

Undefined name `np`

(F821)